dynatrace-aws-logs.sh

```diff
@@ -34,6 +34,7 @@ actions:
   {deploy,subscribe,unsubscribe,discover-log-groups}
                         Actions to be executed by the script:
     deploy              Deploy AWS infrastructure (AWS Kinesis Firehose and AWS Lambda) configured to forward logs from AWS CloudWatch to Dynatrace ActiveGate.
+    delete              Delete AWS infrastructure.
```

Comment:
" Delete AWS infrastructure." sounds a little scary

Reply:
Maybe but IMHO it's consistent with `deploy` description.

Reply:
My proposition:
```
deploy - Deploy AWS log forwarder stack to AWS.
delete - Delete AWS log forwared stack.
```
Actually deploy description is not really even up to date (actually it's my fault ;) ) as there are more elements to this infrastructure now (ec2, vpc, etc. )

Reply:
Sounds good. Updating the description then.

Reply:
Done.